assets/src/edit-story/panels/size/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
```

Comment:
⚠️ `no-unused-vars`

Reply:
Well, not anymore of course :)

Reply:
yes, handlesubmit is on original component now.